components/desktop/wmaker/Makefile

```diff
@@ -0,0 +1,69 @@
+
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL)". You may
+# only use this file in accordance with the terms of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source. A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2021 Gary Mills
+# Copyright (c) 2017 Dariusz Sendkowski
+#
+
+include ../../../make-rules/shared-macros.mk
+
+COMPONENT_NAME=         WindowMaker
+COMPONENT_VERSION=      0.96.0
+COMPONENT_REVISION=     1
+COMPONENT_SUMMARY=      Window Maker is an X11 window manager originally designed to provide integration support for the GNUstep Desktop Environment, although it can run stand alone. In every way possible, it reproduces the elegant look and feel of the NeXTSTEP user interface.
```

Comment:
Please delete COMPONENT_REVISION. Incrementing it is only needed when you need to make changes to the package without changing the COMPONENT_VERSION.